### ISSUE TITLE
Unable to build gazebo_plugins in ROS Kinetic

### ISSUE THREAD

**Issue by mtc-20** [hsrwrobotics/niryo_one_ros#10]
# Solution

- first make sure you have protoc compiler installed

  ```bash
  sudo apt-get install protobuf-compiler
  ``` 
- makse sure to clean your catkin_ws 
  ```bash
  rm -rf ~/catkin_ws/build/ ~/catkin_ws/devel/
  ```

- if you have errors related to mutex or some gazebo cpp codes
  - update your gcc. follow [these instructions](https://gist.github.com/application2000/73fd6f4bf1be6600a2cf9f56315a2d91) 

@quangnhat185

**Comment by quangnhat185** [comment]
> @quangnhat185 does this give you notification mail?

Yes it did. I received two notifications, one from my Github app and one from email

**Comment by mtc-20** [comment]
You have a GitHub app? 😱
